### PR TITLE
Add instance list side panel

### PR DESCRIPTION
**Pull request by edlerd**

### CODE REVIEW

File: static/js/pages/instances/InstanceDetailPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            </a>
+          ))}
+        />
+        <h5 className="u-no-margin--bottom">Networks</h5>
+        <List
+          className="u-no-margin--bottom"
+          items={Object.values(instance.expanded_devices)
+            .filter(isNicDevice)
+            .map((item) =>
+              item.name ? item.name + ": " + item.network : item.network
```
Comment (lorumic) [inline static/js/pages/instances/InstanceDetailPanel.tsx:111]: I propose using a template string here, to make it more readable: ``${item.name}: ${item.network}``

File: static/js/pages/instances/InstanceList.tsx [hunk 2]
```diff
@@ -59,26 +90,34 @@ const InstanceList: FC = () => {
...
     );
 
     return {
+      className: selected === instance.name ? "u-row-selected" : undefined,
       columns: [
+        {
+          content: status,
+          role: "rowheader",
+          className: "u-truncate",
+          "aria-label": "Status",
```
[the remark this answers is not in the document]
Comment (edlerd) [reply static/js/pages/instances/InstanceList.tsx:109]: The API returns `instance.status` so we should stick with that.

File: static/js/pages/instances/actions/DeleteInstanceBtn.tsx [hunk 3]
```diff
@@ -11,20 +11,14 @@ interface Props {
...
 
 const DeleteInstanceBtn: FC<Props> = ({
   instance,
   notify,
   onFinish,
-  appearance = "base",
-  className = "p-contextual-menu__link",
-  isDense = true,
-  label = "Delete instance",
+  isDense,
```
[the remark this answers is not in the document]
Comment (edlerd) [reply static/js/pages/instances/actions/DeleteInstanceBtn.tsx:21]: we don't need the prop in the interface any more.

File: static/sass/styles.scss [hunk 4]
```diff
@@ -70,6 +70,58 @@
   overflow-y: scroll;
 }
 
+.p-instance-detail-panel {
+  border: $border;
+
+  .p-instance-detail-panel--header {
+    border-bottom: $border;
+    padding-left: $sp-xx-small;
```
[the remark this answers is not in the document]
Comment (edlerd) [reply static/sass/styles.scss:78]: true, `$sp-large` looks much better here.

File: static/js/pages/instances/InstanceDetailPanel.tsx (hunk 1 above)
Comment (edlerd) [reply static/js/pages/instances/InstanceDetailPanel.tsx:111]: good idea.

File: static/js/pages/instances/InstanceList.tsx [hunk 5]
```diff
@@ -156,27 +199,97 @@ const InstanceList: FC = () => {
...
+                  label: "All states",
+                  value: "any",
+                },
+                {
+                  label: "Running",
+                  value: "Running",
+                },
+                {
+                  label: "Stopped",
+                  value: "Stopped",
```
[the remark this answers is not in the document]
Comment (edlerd) [reply static/js/pages/instances/InstanceList.tsx]: exported into a dedicated file.

**Review by lorumic — APPROVED** [review]
LGTM

**Pull request merged**